test/extract-js.js

```diff
@@ -0,0 +1,71 @@
+var assert = require('assert');
+var Checker = require('../lib/checker');
+
+describe('modules/extract', function() {
```

Comment:
`modules/extract-js` although `lib/extract-js` would be more accurate (though, I'm aware we have the discrepancy for other test suites).